src/main/java/org/numenta/nupic/Parameters.java

```diff
@@ -23,18 +23,13 @@
 package org.numenta.nupic;
 
 import java.io.IOException;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.EnumMap;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Random;
-import java.util.Set;
+import java.util.*;
```

Comment:
I know we never discussed this, but the standard is to always use explicit "import" statements... rather than wildcards...